Page outlining which controlled vocabulary terms and ontologies we are using in IDR

eleanorwilliams opened the pull request:
Comments welcome.  All the main links to ontologies are to their official sources, but I could make them all OLS beta (easier to view there).

Commit:
First into page about what CVs and ontologies using in IDR

Commit:
moved VocabulariesIntro.md -> README.md

will-moore commented on README.md:
```diff
@@ -1 +1,33 @@
+
+## Dataset descriptors
+
+For high level information describing the type of dataset, e.g. this is a primary RNAi high content screen imaging dataset,  we have put together some terms to describe apects of the High Content Screens and these are now in the [Experimental Factor Ontology](http://www.ebi.ac.uk/efo) (EFO).  For example the following terms (and child terms below them) have been added: 
```
Maybe quotes around the example description: e.g. "This is a primary RNAi high content screen imaging dataset",

will-moore replied:
High Content Screen should always be capitalised (not 'high content screen')?

will-moore commented on README.md:
```diff
@@ -1 +1,33 @@
+ * [screening library](http://www.ebi.ac.uk/ols/beta/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007559)
+ * [HCS protocols](http://www.ebi.ac.uk/ols/beta/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007570)
+
+We are also looking at using terms from the [Biological imaging methods](http://purl.obolibrary.org/obo/fbbi) ontology or getting some terms from this ontology imported into EFO.
```
I wonder if there's a more user-friendly url for fbbi, instead of a directory of files?

bramalingam commented:
This is definitely a good start.

Looking at Screens using a top-down approach:
I was generally searching for a structure like this :

Study(Acquistion && Experimental Protocol)
ImageRawData
ProcessedData
MetaAnalysis(Bulk Annotations)

And all of this is covered in this documentation.

Although everything is there, I felt the landing page did not have a natural flow,Dataset descriptors is definitely a top level header, but the rest could potentially come under a common header? : (Metadata Resources or something on those lines) , which would include sample annotation,control types and cellular phenotypes…

Here are few minor points/add-ons to the controlled vocabulary
Was searching for the following/related terms in the document to understand the specifics of the screen:

Screen Size : Plate Dimensions : X_Y would give the user an idea of the type of plate used (96 : 12_8, 384: 24*16, etc)
Screen Type : Pilot Screen/ Pilot Study /Main Screen
Screen Experiment type : Live Cell Assays, Fixed Cell Assays
Library : Reagents/Library ID : ex: DRSC Genome-wide RNAi Library (DRSC 2.0)
Replicates information: …. For controls/test wells
Plate design : Plate map  ..

Processed data column type : ? Just not sure if this term is easy to understand..
Processed data headers? Would something like this be more generic?

Not sure if this is what was expected out of this review, but am more than willing to have another go, if there is a different scope.

will-moore commented on README.md:
```diff
@@ -1 +1,33 @@
+
+# IDR Controlled Vocabularies and Ontology Annotation
+
+We are using a number of controlled vocabularies and ontologies in the BBSRC-funded [Image Data Repository](http://idr-demo.openmicroscopy.org/webclient/userdata/?experimenter=-1) (IDR) project.  Our approaches to annotation are being modified and expanded as we gather more types of imaging datasets, expand the functionality we are providing within OMERO for the IDR,  and as a result of discussio ...
```
The controlled vocabulary and ontology sources spreadsheet looks like it's for "internal use only" and not intended for external info? E.g. In most table cells the content is too long so I have to look at the top of the page to read the text in the currently selected cell. And the layout is quite haphazard etc. I wonder if (in due course) this could be moved into a more readable format, E.g. a markdown file that simply used a hierarchical bullet points for each Tag, with nested terms (linking to their ontologies)?

will-moore commented:
Minor comment: It might be easier to work with .md files in github etc if you keep the lines shorter? Just looking at the comments above in github, there is a ton of text highlighted for each comment, because the comment applies to a very long line. Also, changes to single lines will show a much smaller diff if the lines are not too long.

eleanorwilliams commented:
Thanks for the comments @will-moore and @bramalingam!  I'll try and address them tomorrow.  The spreadsheet is really just my reminders of what to terms to reuse in different parts of the study files I'm creating, so maybe I need to make more concise version for external consumption with only the fields I talk about in the document.   

Never thought about shortening the lines, but its a good point, and I will do.

Commit:
Added more paragraphing, changed section headers, changed all ontology links to OLS/beta, added description of processed data file CV

Commit:
Some more minor edits after a read through

Commit:
another typo fixed, and removed the {:target="_blank"} statements that dont work

Commit:
added text about Experimental Conditions

eleanorwilliams commented:
New version ready for review. This page is intended to be shown to other groups working on controlled vocabularies and ontologies.  It will eventually link to other pages introducing the IDR and will need modification when they exist to alter the first paragraph. But for now it is a stand alone page to give a brief outline of the kinds of things we are doing.   

I've simplified the spreadsheet with terms in now, and have tried to cover everything mentioned in the spreadsheet in this .md file.   I don't want to go into details about how we are storing all this info yet, as our templates are not really ready for public consumption, but we want to share with others what we are doing so far about using controlled vocabs and ontologies. 

Also I say at the end that we welcome comments, but don't tell them how to.  I've left it like that because we don't have a dedicated email address for IDR correspondence, and also we are likely to be telling specific people to look at this page ourselves, so they can contact directly. 

One thing, I couldn't figure out how to make the links open in a new tab.  If you know how to do this can you let me know.

hflynn commented on README.md:
```diff
@@ -1 +1,42 @@
+
+For example the following terms (and child terms below them) have been added:
+
+ * [high content screen](http://www.ebi.ac.uk/ols/beta/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007550)
```
high content screen is written as High Content Screen elsewhere, also where you use HCS instead seems a bit inconsistent

hflynn commented on README.md:
```diff
@@ -1 +1,42 @@
+We are also looking at using terms from the [Biological imaging methods](http://www.ebi.ac.uk/ols/beta/ontologies/fbbi) ontology or importing them into EFO.
+
+
+### Samples, Experimental Variables and Library Annotation
```
Subheading above only has a starting capital

hflynn commented on README.md:
```diff
@@ -1 +1,42 @@
+
+### Samples, Experimental Variables and Library Annotation
+
+For sample annotation we are using [NCBITaxon](http://www.ebi.ac.uk/ols/beta/ontologies/ncbitaxon) for organisms, and EFO so far for cell lines.  EFO should also cover most sample characteristics when we move to other types of imaging data with perhaps more detailed sample information.  It also provides a description of the experimental variables or conditions we have seen so far such as environm ...
```
Not a fan of the repeated use of 'so far' in this paragraph, don't think you really need either of them given the context and certainly the first one doesn't read very nicely.

pwalczysko replied:
Want to add a link to EFO ?

Edit: Sorry, I see the link is there couple of lines above.

hflynn commented:
@eleanorwilliams a few very minor nitpicks.
Re: feedback - people can always open a GH issue anyway.

My web developer friend tells me it is poor practice to enforce where links open in your coding and you should allow people to choose via their browser options instead. I don't think GH flavoured markdown supports it.

pwalczysko commented on README.md:
```diff
@@ -1 +1,42 @@
+## IDR Controlled Vocabularies and Ontology Annotation
+
+We are using a number of controlled vocabularies and ontologies in the BBSRC-funded [Image Data Repository](http://idr-demo.openmicroscopy.org/webclient/userdata/?experimenter=-1) (IDR) project.  Our approaches to annotation are being modified and expanded as we gather more imaging datasets, expand the functionality we are providing within OMERO for the IDR,  and as a result of discussions with o ...
+
```
Maybe specify who are `others` ? (community ?, imaging community ? data storage community ? similar projects ?)

pwalczysko commented:
Links work. Looks good.

Commit:
fixed first section heading, high content screen casing, who "others" are, removed "so far"s

eleanorwilliams commented:
1. Fixed high content screen/High Content Screen elsewhere
2. Fixed subheadings so all have same casing
3. Removed the 'so far's.
4. Changed 'others' to 'others in the imaging community' 

Thanks for the comments.

hflynn commented:
Looks good :+1:

pwalczysko commented:
Fine by me.

Commit:
added example of terms added to CMPO, and info about how many terms added since IDR started

eleanorwilliams commented:
Jason says it looks ok in https://trello.com/c/PndcSPsg/244-set-up-blog-website-for-vocabularies-and-ontologies-used.  Merging.
